lib/token-verifier/WP_Auth0_SymmetricVerifier.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
[See this diff](https://github.com/auth0/auth0-PHP/pull/401/files#diff-25934a723f93d7eb12e57c7f8dcf53fe) for changes from approved PHP SDK ones.